peps/pep-0777.rst

```diff
@@ -0,0 +1,304 @@
+PEP: 777
+Title: Re-inventing the Wheel
```

Comment:
"How to re-invent the wheel" might be a better title, as this PEP does not actually re-invent the wheel; it just makes it possible for a companion PEP to do the re-inventing. (However, feel free to ignore this suggestion.)

Reply:
Good point, I like your suggestion.